### ISSUE TITLE
Processing script - improvements

### ISSUE THREAD

**Issue by ksedivyhaley** [UBC-MDS/DSCI_522_Group313#51]
- Splits too late - you need to split data, and then create a scaling transformer for the training set, save that and then apply that to the test set. Otherwise your test set is influencing your training data. 
- You have a neat idea to derive the directory for the split data from the store_path, but this isn't well documented.

**Comment by suvarna-m** [comment]
Hi Kate, 
Thanks so much for your feedback. As discussed in lab, we did not transform the data until after it was split, but to mitigate this, I moved the code chunk where we call in the transformer to **after** the split. The README has been updated to better document the store_path split.
